docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -0,0 +1,153 @@
+    | Boolean    | `true` or `false`. Values that evaluate to `true` or `false`, such as 1 and 0, are not accepted.       |
+    | Version    | Version value that follows the [Semantic Versioning](https://semver.org/) convention, such as `x.y.z`. |
+    | IPv4       | Valid representation of an IPv4 address.                                                               |
+    | IPv4 CIDR  | A CIDR block of IP addresses that follow the IPv4 standard.                                            |
+    | IPv6       | Valid representation of an IPv6 address.                                                               |
+
+8.  Optionally, configure additional data validation. The following table describes the available parameters.
+
+    | **Parameter**           | **Description**                                                                                                                                                                                                                           |
+    | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | Custom input validation | Whether the input is validated against a regular expression. Based on the [regexp](https://pkg.go.dev/regexp) engine.                                                                                                                     |
+    | Required                | Whether the input is required.                                                                                                                                                                                                            |
+    | Default value           | Whether the variable has a default value.                                                                                                                                                                                                 |
+    | Mask value              | Whether the input is masked with asterisks in the layer YAML configuration. When you export a profile with masked variables, they will be masked in the exported profile. Upon import, you will be required to provide the masked values. |
+    | Hidden                  | Whether the variable is hidden during cluster deployment.                                                                                                                                                                                 |
+    | Read-only               | Whether the variable is editable during cluster deployment.                                                                                                                                                                               |
```

Comment:
These two, `read-only` and `hidden`  two also apply to the cluster profile creation process.